yacc/main.c

```diff
@@ -138,7 +138,7 @@ void done(int k)
 
 void onintr(int dummy)
 {
-    done(1);
+    done(128 + dummy);
```

Comment:
That may well be so, but that's definitely not something to be smuggled inside another patch. Could this change at least be put on its own (I'm not entirely convinced that that belongs in this pull request at all)